### ISSUE TITLE
Completion Certificates for studies

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#1446]
# TL;DR
Parents should be able to download a completion certificate for each study in their /studies/history page, produced automatically by Lookit.

# Narrative

Some parents very much like completion certificates, and some researchers go through the labor of providing a customized one for each family. This could be replaced with an automated process based on the study history page. Parents can already see a lot of information about each study their child has done on [the history page](https://childrenhelpingscience.com/studies/history/). A screenshot is below of what it already looks like (though a real session would likely have the child visible instead of me!). It would be great if there were a "download completion certificate" button that generated a PDF with the information. We could think about the ideal information and format and whether a still frame of the video should be included (if so, allowing the parent to select it would be useful). Probably something like child name, study name, the picture representing the study, date, and the lab/university doing the study. Maybe an option to have the short description included. 

![Screen Shot 2024-07-26 at 9 23 08 AM](https://github.com/user-attachments/assets/3b654cc3-f059-4464-b4ce-8f163511f835)
